core/src/test/java/org/hibernate/ogm/backendtck/inheritance/tableperclass/family/TablePerClassInheritancePersistTest.java

```diff
@@ -68,7 +69,7 @@ public void testPersistEntititesWithoutErrors() {
 	@Test
 	@TestForIssue(jiraKey = "OGM-1294")
 	@SkipByDatastoreProvider(
-			value = { MONGODB, NEO4J_BOLT, NEO4J_HTTP, NEO4J_EMBEDDED },
+			value = { MONGODB, NEO4J_BOLT, NEO4J_HTTP, NEO4J_EMBEDDED, INFINISPAN_REMOTE },
```

Comment:
I chose to add test skips at the beginning in order to avoid having commits that would not pass test. But as you prefer.

Reply:
> I chose to add test skips at the beginning in order to avoid having commits that would not pass test. But as you prefer.

There is a misunderstanding, when I tried t orun the build for this commit there was a failure. Maybe it wasn't caused by this particular change but something in this commit is not right.

It's not about what I prefer, it just that it seems there is a problem, if you think that I'm wrong and the build pass with this commit I will check again.

Reply:
@DavideD, you were right. I'm going to fix the commit.